### ISSUE TITLE
[Question] about Getting confidence score

### ISSUE THREAD

**Issue by phamkhactu** [DevashishPrasad/CascadeTabNet#184]
Thanks for your  excellent work.

I want to get best confidence score in result return from model. I don't see any  docs or detail example from it.

```
result = inference_detector(model, img)

```

How can I get conf for each table?

**Comment by qminh369** [comment]
Helllo, Can you help me to run CascadeTabNet in newest version? I have some conflict torch and mmcv version.

**Comment by phamkhactu** [comment]
> Helllo, Can you help me to run CascadeTabNet in newest version? I have some conflict torch and mmcv version.

Yes, I ran it okie. You should use docker image to avoid conflict version.
or here is my version:
```
mmcv==1.5.3 mmcv-full
mmdet==2.25.0
```

**Comment by parthikjoshi** [comment]
I tried so many times different methods but not able to set up cascadetabnet. Are there any other  libraries for detecting tables from scanned PDFs and images?
